src/templates/_includes/nav.html

```diff
@@ -0,0 +1,44 @@
+<mkt-nav-root>
```

Comment:
You defined this to inherit from `ul`, have you tried making this `<ul is="mkt-nav-root">`? I guess that probably messes with the styles though.